### PR TITLE
Modify HelpCommand to take keyword of command

### LINKED ISSUE #181: Update HelpCommand
Update help command to handle specific commands

Command format:
help KEYWORD

### PR DESCRIPTION
**Pull request by HynRidge**
Add HelpCommandParser to parse user argument
Add MESSAGE_USAGE in ListCommand,RedoCommand,UndoCommand,ExitCommand,ClearCommand.
Add unit test for HelpCommand and HelpCommandParser

Note: Safe to merge!!

### CODE REVIEW

**Review by luo-git — APPROVED** [review]
Code is quite clean. I like it. LGTM!

Edit: Thanks for cleaning up the help message format too.

File: src/main/java/seedu/address/logic/commands/HelpCommand.java [hunk 1]
```diff
@@ -13,19 +16,69 @@
...
             + String.format("%-22s %s", ExitCommand.COMMAND_WORD, ExitCommand.EXIT_COMMAND_USAGE + "\n")
             + String.format("%-22s %s", FindCommand.COMMAND_WORD, FindCommand.FIND_MESSAGE_USAGE + "\n")
             + String.format("%-24s %s", ListCommand.COMMAND_WORD, ListCommand.LIST_MESSAGE_USAGE + "\n")
             + String.format(STRING_FORMAT, OpenCommand.COMMAND_WORD, OpenCommand.OPEN_MESSAGE_USAGE + "\n")
+            + String.format("%-21s %s", RedoCommand.COMMAND_WORD, RedoCommand.REDO_COMMAND_USAGE + "\n")
             + String.format("%-21s %s", RetagCommand.COMMAND_WORD, RetagCommand.RETAG_MESSAGE_USAGE + "\n")
             + String.format(STRING_FORMAT, ShowCommand.COMMAND_WORD, ShowCommand.SHOW_MESSAGE_USAGE + "\n")
             + String.format("%-22s %s", TagCommand.COMMAND_WORD, TagCommand.TAG_COMMAND_USAGE + "\n")
-            + String.format(STRING_FORMAT, UntagCommand.COMMAND_WORD, UntagCommand.UNTAG_MESSAGE_USAGE + "\n")
-            + String.format(STRING_FORMAT, LabelCommand.COMMAND_WORD, LabelCommand.LABEL_COMMAND_USAGE + "\n");
```
Comment (luo-git) [inline src/main/java/seedu/address/logic/commands/HelpCommand.java:25]: Is label command left out?

Comment (HynRidge) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java:25]: Owh yes will add that in

Comment (luo-git) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java:25]: Nice after that can merge alr

Comment (HynRidge) [reply src/main/java/seedu/address/logic/commands/HelpCommand.java:25]: Added label command

**Review by luo-git — APPROVED** [review]

**Pull request merged**